in_toto/in_toto_mock.py

```diff
@@ -31,8 +31,21 @@
   in-toto-mock --name write-code -- touch foo.py
   ```
 
-"""
+<Arguments>
+  name:
+          A unique name to relate mock link metadata with a step defined
+          in the layout.
+  command:
```

Comment:
The `command` argument in this docstring might be confusing to some users.  On the one hand, `--name` is an actual command-line argument, but `--command` is not.  That is, `$ in_toto_mock.py --name foo touch bar.log` is the expected format, where `touch bar.log` are positional arguments.

Reply:
Well technically, both are arguments, one *optional* (which I find confusing because it's actually required, see b19b17fdd4865896f70fb4f4ee8f8ebcb081d11d) and one *positional*.

Also, I think it's fine, because as you mentioned above, the user will probably not look at the docstring but rather at the command's help message, which should make the usage clear:
```
$ in-toto-mock -h
usage: in-toto-mock [-h] --name <unique step name> -- <command> [args]

Executes link command and records metadata

positional arguments:
  <command>             Link command to be executed with options and arguments

optional arguments:
  -h, --help            show this help message and exit

required named arguments:
  -n <unique step name>, --name <unique step name>
                        Unique name for link metadata
```